packages/kbn-apm-synthtrace/src/lib/apm/span.ts

```diff
@@ -57,39 +57,33 @@ export function httpExitSpan({
   const spanType = 'external';
   const spanSubType = 'http';
 
-  return [
+  return {
     spanName,
     spanType,
     spanSubType,
-    {
+    apmFields: {
       'destination.address': destination.hostname,
       'destination.port': parseInt(destination.port, 10),
       'service.target.name': destination.host,
       'span.destination.service.name': destination.origin,
       'span.destination.service.resource': destination.host,
       'span.destination.service.type': 'external',
     },
-  ];
+  };
 }
 
-export function dbExitSpan({
-  spanName,
-  spanSubType,
-}: {
-  spanName: string;
-  spanSubType?: string;
-}): [string, string, string | undefined, ApmFields] {
+export function dbExitSpan({ spanName, spanSubType }: { spanName: string; spanSubType?: string }) {
   const spanType = 'db';
 
-  return [
+  return {
     spanName,
     spanType,
     spanSubType,
-    {
+    apmFields: {
       'service.target.type': spanSubType,
       'span.destination.service.name': spanSubType,
       'span.destination.service.resource': spanSubType,
       'span.destination.service.type': spanType,
     },
-  ];
+  };
```

Comment:
If you destructure `apmFields` as mentioned above, this becomes:

```ts
  return {
    spanName,
    spanType,
    spanSubType,
    'service.target.type': spanSubType,
    'span.destination.service.name': spanSubType,
    'span.destination.service.resource': spanSubType,
    'span.destination.service.type': spanType,
  ];
 
```

Which I think looks better.